app/views/content/train-to-be-a-teacher/other-training-you-can-get.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Other training and experience you can get
```

Comment:
Does this section include training? Will this confuse users to think this is another form of teacher training?
Maybe we could change this to 'Get experience to find out if teaching is right for you'

Reply:
There's also some text underneath this card which I can't find in GitHub, which says: 'whatever your situation there's a training programme for you...'
This might confuse users as the teaching internship and school experience isn't a training programme. 
We could change that to 'whatever your situation there's an opportunity to experience what it's a like to be a teacher'